INPTP_AppForFixing/EmployeeDialog.cs

```diff
@@ -71,9 +89,30 @@ private bool CheckValidInputForm()
             int id;
             double salary;
 
-            if (Int32.TryParse(tBID.Text, out id) && Double.TryParse(tBSalary.Text, out salary) && tBID.Text.Length > 0 &&
-                tBFirstName.Text.Length > 0 && tBJob.Text.Length > 0 && tBLastName.Text.Length > 0) return true;
-            return false;
+            return Int32.TryParse(tBID.Text, out id) && 
+                Double.TryParse(tBSalary.Text, out salary) && 
+                tBID.Text.Length > 0 &&
+                tBFirstName.Text.Length > 0 && 
+                tBJob.Text.Length > 0 && 
+                tBLastName.Text.Length > 0;
+           
+        }
+
+        private bool isIdUniqui()
```

Comment:
According to .net naming guidelines this should be IsIdUnique or IsIDUnique, but the first version is better